### PR TITLE
Add new platform RHV support for testing

### PR DESCRIPTION
**Pull request by pkesavap**
This commit has base structure for  RHV platform in ocs-ci 
signed-off-by: Prajith Kesava Prasad <pkesavap@redhat.com>

### CODE REVIEW

**Comment by pkesavap** [comment]
Hi @ebenahar can you review now?

File: ocs_ci/ocs/platform_nodes.py [hunk 1]
```diff
@@ -59,6 +60,15 @@ def get_nodes_platform(self):
         return self.cls_map[platform]()
 
 
+class RHVNodes(object):
```
Comment (ebenahar) [inline ocs_ci/ocs/platform_nodes.py:63]: For better ordering, let's place this class below the base class of `NodesBase`

Comment (pkesavap) [reply ocs_ci/ocs/platform_nodes.py:63]: @ebenahar done, have moved RHVNodes class below the class of NodesBase

**Comment by petr-balogh** [comment]
I think we already have this RHV IPI handled in different PR as I see the code already in master:
https://github.com/red-hat-storage/ocs-ci/blob/1601ce72fb9932b712829c7b9f8e30349f9c759c/ocs_ci/deployment/rhv.py#L87

**Pull request closed without merging**